typescript/infra/scripts/check-helloworld.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  HelloWorldApp,
+  HelloWorldChecker,
+  HelloWorldConfig,
+} from '@abacus-network/helloworld';
+import {
+  HelloWorldContracts,
+  helloWorldFactories,
+} from '@abacus-network/helloworld/dist/sdk/contracts';
```

Comment:
I was going to fix this in a follow-up PR but `sdk` isn't really a good name for what that folder is right? Feel free to ignore for this PR, just thinking out loud

Reply:
Making a note for myself, will circle back on this